src/lib/libdwarf/libdwarf.h

```diff
@@ -50,24 +50,22 @@
 #undef DW_API
 #endif /* DW_API */
 
-#ifndef LIBDWARF_STATIC
-# if defined(_WIN32) || defined(__CYGWIN__)
-#  ifdef LIBDWARF_BUILD
-#   define DW_API __declspec(dllexport)
-#  else /* !LIBDWARF_BUILD */
-#   define DW_API __declspec(dllimport)
-#  endif /* LIBDWARF_BUILD */
-# elif (defined(__SUNPRO_C)  || defined(__SUNPRO_CC))
-#  if defined(PIC) || defined(__PIC__)
-#   define DW_API __global
-#  endif /* __PIC__ */
-# elif (defined(__GNUC__) && __GNUC__ >= 4) || \
+#if defined(_WIN32) || defined(__CYGWIN__)
```

Comment:
The DWAPI macro needs to be set for static builds too, otherwise I run into reference to undefined symbol errors

Reply:
Comment now outdated, nothing needed changing here